<launch>

    <node pkg="Robotics_project1" type="vel_node" respawn="false" name="vel" output="screen"/>
    <node pkg="Robotics_project1" type="odom_node" respawn="false" name="odom" output="screen"/>

</launch>
